From ace040c02f2e059d8d02662524cee2709ca025f2 Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Fri, 20 Oct 2017 13:41:55 +0300
Subject: [PATCH 069/706] enetc: Update driver version to v0.4

This is the first driver version supporting m0165 simulator model,
with many new features:
* VF mac address configuration ("ip link set vf mac" command)
* VF VLAN isolation ("ip link set vf vlan" command)
* PF MAC filtering and initial control ring support
- supports MAC unicast and multicast, exact match and hash table
  filters
- implements set_rx_mode()
* Configurable MTU and jumbo frame on Tx enabled
* Initial ethtool statics support ("ethtool -S")
* Initial Receive Flow Steering (RFS) support:
- flow classification based on L2 and IPv4 addresses, UDP4 (over
  IPv4), TCP4, SCTP4 field:
- implements "ethtool -n" and "ethtool -N flow-type" (common
  filering options, rule insertion and deletion)

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit cf661c9adc1bde427586221bd88ab2e8f975daaf)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc.c b/drivers/net/ethernet/freescale/enetc/enetc.c
index c3a04638d099..48c03a271c10 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc.c
@@ -3,7 +3,7 @@
 #include "enetc.h"
 
 #define ENETC_DRV_VER_MAJ 0
-#define ENETC_DRV_VER_MIN 3
+#define ENETC_DRV_VER_MIN 4
 
 #define ENETC_DRV_VER_STR __stringify(ENETC_DRV_VER_MAJ) "." \
 			  __stringify(ENETC_DRV_VER_MIN)
-- 
2.17.1

